moped-editor/src/views/dev/LookupsView/RecordTable.js

```diff
@@ -14,7 +14,7 @@ import CircularProgress from "@mui/material/CircularProgress";
  * @returns { JSX } an MUI TableCell component with the row/column value
  */
 const WrappedTableCell = ({ row, column: { key, handler } }) => {
-  return <TableCell>{handler ? handler(row[key]) : row[key]}</TableCell>;
+  return <TableCell>{handler ? handler(row[key], row) : row[key]}</TableCell>;
```

Comment:
I added the row object into the handler call, but only one of our handler functions uses `row`, the one I just wrote. 

I can't decide if i should just leave the rest of the handler function definitions as is, or if i should update all of them to make explicit that they are getting an unneeded row object.

Reply:
hmmm, i'm going to read about this some. I think part of why this feels off is that some of the `handler` functions are returning JSX so they are function components in themselves. 🤔

Reply:
this looks great to me! nbd to omit the `row` prop from the other handlers that do not access it. i think that is preferable to adding, for example, the `row` arg to the `workTypeHandler` function signature 🚢

Reply:
yep, i was getting side-tracked with code beyond this diff. thanks for reining this back in, John! 🙏